notebooks/Adapter_Whisper_Audio_FineTuning.ipynb

```diff
@@ -0,0 +1,555 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fine Tuning Whisper using Adapters\n",
+    "\n",
+    "In this tutorial, we will be demonstrating how to fine-tune a [Whisper](https://arxiv.org/abs/2212.04356) model using adapters. We will be adding [LoRA adapters](https://docs.adapterhub.ml/methods#lora) to Whisper while freezing the model weights. In addition, we will incorporate a sequence to sequence head on top of the model so that we can do speech recognition. This tutorial is widely inspired by the [Whisper PEFT-Lora blog post](https://github.com/huggingface/peft/blob/main/examples/int8_training/peft_bnb_whisper_large_v2_training.ipynb), be sure to check it out for more details on ``Whisper`` fine-tuning. \n",
+    "\n",
+    "For more information on the Whisper Model, please visit the huggingface model card https://huggingface.co/openai/whisper-large-v3"
+   ]
```

Comment:
```suggestion
    "In this tutorial, we will be demonstrating how to fine-tune a [Whisper](https://arxiv.org/abs/2212.04356) model using adapters. We will be adding [LoRA](https://docs.adapterhub.ml/methods#lora) to Whisper and will incorporate a sequence to sequence head on top of the model so that we can do speech recognition. This tutorial is build on this [Whisper PEFT-Lora blog post](https://github.com/huggingface/peft/blob/main/examples/int8_training/peft_bnb_whisper_large_v2_training.ipynb). \n",
    "\n",
    "For more information on the Whisper Model, please visit the [Hugging Face model card](https://huggingface.co/openai/whisper-large-v3) or see the original [OpenAI Blog Post](https://openai.com/index/whisper/)."
   ]
```